### ISSUE TITLE
Distinguish between left and right hands？

### ISSUE THREAD

**Issue by wangjye777** [facebookresearch/frankmocap#155]
How to distinguish left and right hands for ego_centric ? Looking forward to your reply

**Comment by wangjye777** [comment]
# hand side info (left/right)
 lr_vector = loss_list[2][0].detach()
 lr = torch.sigmoid(lr_vector) > 0.5
 lr = lr.squeeze(0).float()



Does the value of lr judge left and right hands?

**Comment by penincillin** [comment]
@wangjye777 
We adopt trained hand detectors from [100DOH](https://fouheylab.eecs.umich.edu/~dandans/projects/100DOH/) to distinguish left/right hands. Please refer to [Ego_Centric_Detector](https://github.com/facebookresearch/frankmocap/blob/46de6e745e2c69d130bf5b5f1e1fb1ccc333cfa7/handmocap/hand_bbox_detector.py#L136) for more details.

**Comment by wangjye777** [comment]
> @wangjye777 We adopt trained hand detectors from [100DOH](https://fouheylab.eecs.umich.edu/~dandans/projects/100DOH/) to distinguish left/right hands. Please refer to [Ego_Centric_Detector](https://github.com/facebookresearch/frankmocap/blob/46de6e745e2c69d130bf5b5f1e1fb1ccc333cfa7/handmocap/hand_bbox_detector.py#L136) for more details.
